docs/compatibility.md

```diff
@@ -164,6 +164,10 @@ similar issue exists for writing dates as described
 appears to work for dates after the epoch as described
 [here](https://github.com/NVIDIA/spark-rapids/issues/140). 
 
+The plugin supports reading `uncompressed`, `snappy` and `zlib` ORC files and writing `uncompressed`
+ and `snappy` ORC files.  At this point, the plugin does not have the ability to fall back to the 
+ CPU when reading or writing an unsupported compression format, and will error out in that case. 
```

Comment:
Sorry it is just reading.  Writing we will fall back to the CPU for an unsupported format.

Reply:
Corrected to only indicate reading will error out.